src/util/indicators_parser.py

```diff
@@ -0,0 +1,288 @@
+import csv
+import json
+import sys
+import xlrd
+class IndicatorsParser(object):
```

Comment:
@p00j4 can you run the file with pep8 or flake8 and fix those syntatical errors ?